opentok/opentok-meet
# Remove custom stylesheet and replace with preset. [OPENTOK-43672]

#### What is this PR doing?
Meet archiving uses a custom built dynamic CSS layout which displays either a standard bestfit (of all those streams which are not video-source-type screen) or a bestfit of screenshares (if 1 or more is present ). 
There is now a preset available which does exactly that. So we can remove the custom CSS. 

#### How should this be manually tested?
Create an archive and test e.g. with 3 camera feeds. Then add 3 screenshares, then remove them one by one. 
In the output video you should see a bestfit of camera streams until you share the first screen, then it becomes a bestfit 
of screenshares until you remove the last screen and finally it reverts to a bestfit of camera streams. 

#### What are the relevant tickets?
